src/test/java/com/github/cameltooling/lsp/internal/util/RouteTextBuilder.java

```diff
@@ -16,11 +16,23 @@
  */
 package com.github.cameltooling.lsp.internal.util;
 
+import org.eclipse.lsp4j.Position;
+
 public class RouteTextBuilder {
 	
 	public static final String XML_PREFIX_FROM = "<from uri=\"";
 	private static final String XML_SUFFIX_FROM_SPRING = "\" xmlns=\"http://camel.apache.org/schema/spring\"/>\n";
 	private static final String XML_SUFFIX_FROM_BLUEPRINT = "\" xmlns=\"http://camel.apache.org/schema/blueprint\"/>\n";
+
+	private static final String JAVA_CLASS_BLUEPRINT_CAMEL_ROUTEBUILDER_IMPORT
+			= "import org.apache.camel.builder.RouteBuilder;";
+
+	private static final String JAVA_CLASS_BLUEPRINT_CLASS_DECLARATION = "public class TestRoute";
+
+	private static final String JAVA_CLASS_BLUEPRINT_CAMEL_ROUTEBUILDER_EXTEMD = "extends RouteBuilder";
+
+	private static final String JAVA_CLASS_BLUEPRINT_CONFIGURE_METHOD_DECLARATION = "public void configure()";
```

Comment:
I think the terminology `Bluerpint` should not be used here.

it will collides with the "[Camel XML blueprint namespace](https://camel.apache.org/components/2.x/others/blueprint.html)" which is related to the [Apache Aries Blueprint implementation](https://aries.apache.org/documentation/modules/blueprint.html)